Syntaxes/Second Life LSL.tmLanguage

```diff
@@ -109,7 +109,7 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>\b\w+ *(?=\()</string>
+			<string>\b\w+\s*(?=\()</string>
```

Comment:
This is still a good change as it would allow tab characters here, but it doesn't do what you think. Each regex rule can only see a single line at a time, so matching a function with parens on the next line can't work. Rules can span multiple lines with begin/end but each regex match is a single line.